bundles/org.openhab.binding.verisure/README.md

```diff
@@ -51,16 +51,16 @@ To enable DEBUG logging for the binding, login to Karaf console and enter:
 
 `openhab> log:set DEBUG org.openhab.binding.verisure`
 
-## Supported Things and Channels 
+## Supported Things and Channels
 
-### Verisure Bridge 
+### Verisure Bridge
 
 #### Configuration Options
 
-*   `username` - The username used to connect to http://mypage.verisure.com
+*   `username` - The username used to connect to https://mypage.verisure.com
```

Comment:
I think this url should be:
```suggestion
*   `username` - The username used to connect to https://mypages.verisure.com
```

Reply:
Ok. I have not checked all of them on a functional base.
The main website supported https and i changed all of them. :)